File: publlic_html/CPP/funkcje.cpp
/*
 * szkielet.cpp
 */


#include <iostream>

using namespace std;

int suma(int a, int b)
{
   return a + b;
}

int roznica(int a, int b)
{
   return a - b;
}

int iloczyn(int a, int b)
{
   return a * b;
}

int iloraz(int a, int b)
{
   return a / b;
}

int main(int argc, char **argv)
{

    int a, b;  // deklaracja zmiennych
    a = b = 0; //inicjalizacja zmiennych
    
    cout << "Podaj liczbę: ";
    cin >> a;
    
    cout << "Podaj drugą liczbę: ";
    cin >> b;
    
    cout << a << " " << b;
    
    cout << endl << " Suma: " << suma(a, b) << endl;
    cout << endl << " Róznica: " << roznica(a, b) << endl;
    cout << endl << " Iloczyn: " << iloczyn(a, b) << endl;
    cout << endl << " Iloraz: " << iloraz(a, b) << endl;
    
    
    //DRY-don't repeat yourself
    
	return 0;
}

File: publlic_html/python/funkcje.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wejscie-wyjscie1.py
#  duck typing
#  wszystko co pobierane jest z domyślnego wejścia, tj. terminal, jest znakiem
#  zasięg zmiennych: zasięg lokalny  (ograniczony np. funkcją)


def suma(a, b):
    return a + b
    
    
def roznica(a, b):
    return a - b
    
    
def iloczyn(a, b):
    return a * b
    
    
def iloraz(a, b):
    return a / b

def main(args):
    a = int(input("Podaj pierwszą liczbę: "))
    print(a)
    b = int(input("Podaj drugą liczbę: "))
    print(b)
    
    print("Suma: ", suma(a, b))
    print("Różnica: ", roznica(a, b))
    print("Iloczyn: ", iloczyn(a, b))
    print("Iloraz: ", iloraz(a, b))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
